### PR TITLE
Add gold test for remap config .include directive.

### PR DESCRIPTION
**Pull request by ywkaras**
Included checking for whether traffic_ctl config reload handles .include properly.

### CODE REVIEW

**Review by bneradt — APPROVED** [review]
Looks good to me.

File: tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Test.ContinueOnFail = False
+
+Test.Setup.Copy("wait_reload.sh")
+
+# Define ATS and configure
+ts = Test.MakeATSProcess("ts", command="traffic_manager", enable_cache=False)
+
+ts.Disk.File(ts.Variables.CONFIGDIR + "/test.inc", id="test_cfg", typename="ats:config")
+ts.Disk.test_cfg.AddLine(
+    "map http://example.two/ http://yada.com/ " +
```
Comment (maskit) [inline tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py:31]: yaba.com ?

Comment (ywkaras) [reply tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py:31]: https://www.phrases.org.uk/meanings/yada-yada.html

**Review by randall — APPROVED** [review]

File: tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py (hunk 1 above)
Comment (ywkaras) [reply tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py:31]: It's just a place-holder for a URL that is not really used.

**Comment by zwoop** [comment]
Cherry-picked to v9.1.x branch.

**Pull request merged**